03 Writing Algorithms/03 Securities/06 Filtering Data/04 Default Behavior.html

```diff
@@ -0,0 +1 @@
+<p>The default data filtering model of Equity is the <code>EquityDataFilter</code>, while being <code>ForexDataFilter</code> and <code>SecurityDataFilter</code> for Forex and other Security types. All these default data filter would accept all incoming raw data unfiltered. To view the implementation of the <code>SecurityDataFilter</code> model, see the <a rel='nofollow' target="_blank" href='https://github.com/QuantConnect/Lean/blob/master/Common/Securities/SecurityDataFilter.cs'>LEAN GitHub repository</a>.</p>
```

Comment:
Let's format this into a table.

> The following table shows the default data filter for each security type:

The table should include all of the data filter, including the `IndexDataFilter` even though it doesn't override the `Filter` method

> None of the preceding data filters filter out any data.